Content.Server/Disposal/Unit/EntitySystems/DisposalUnitSystem.cs

```diff
@@ -9,15 +10,22 @@ public override void Initialize()
         {
             base.Initialize();
 
-            SubscribeLocalEvent<DisposalUnitComponent, PhysicsBodyTypeChangedEvent>(BodyTypeChanged);
+
+            SubscribeLocalEvent<DisposalUnitComponent, AnchoredEvent>(OnAnchored);
+            SubscribeLocalEvent<DisposalUnitComponent, UnanchoredEvent>(OnUnanchored);
+
+
+            
```

Comment:
remove this extra whitespace

Reply:
And readd the newline after the closing bracket.

Reply:
Like so?